controller/persistence/posture_check_type_store.go

```diff
@@ -0,0 +1,151 @@
+/*
+	Copyright NetFoundry, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package persistence
+
+import (
+	"github.com/openziti/foundation/storage/boltz"
+	"go.etcd.io/bbolt"
+)
+
+const (
+	FieldPostureCheckTypeOperatingSystems = "operatingSystems"
+)
+
+type PostureCheckType struct {
+	boltz.BaseExtEntity
+	Name             string
+	OperatingSystems []OperatingSystem
```

Comment:
if someone adds the same OS multiple times will there be any consequence? is there any validation for that (i am making this comment at the top of the file, so if it becomes apparent as i read on well - just ignore this comment then) 👍

Reply:
It will take the last one in the list.